systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificIsolatedST.java

```diff
@@ -193,6 +207,97 @@ void testLoadBalancerSourceRanges(ExtensionContext extensionContext) {
+
+        // specify explicit namespace for RoleBindings
+        strimziClusterOperator020Namespaced.getMetadata().setNamespace(namespaceWhereCreationOfCustomResourcesIsApproved);
+        strimziClusterOperator022LeaderElection.getMetadata().setNamespace(clusterOperator.getDeploymentNamespace());
+
+        // reference Cluster Operator service account in RoleBindings
+        strimziClusterOperator020Namespaced.getSubjects().stream().findFirst().get().setNamespace(clusterOperator.getDeploymentNamespace());
+        strimziClusterOperator022LeaderElection.getSubjects().stream().findFirst().get().setNamespace(clusterOperator.getDeploymentNamespace());
+
+        final List<RoleBinding> roleBindings = Arrays.asList(
+                strimziClusterOperator020Namespaced,
+                strimziClusterOperator022LeaderElection
+        );
+
+        // ---- c) defining ClusterRoleBindings
+        final List<ClusterRoleBinding> clusterRoleBindings = Arrays.asList(
+            ClusterRoleBindingTemplates.getClusterOperatorWatchedCrb(clusterOperator.getClusterOperatorName(), clusterOperator.getDeploymentNamespace()),
+            ClusterRoleBindingTemplates.getClusterOperatorEntityOperatorCrb(clusterOperator.getClusterOperatorName(), clusterOperator.getDeploymentNamespace())
+        );
+
+        clusterOperator.unInstall();
+        // create namespace, where we will be able to deploy Custom Resources
+        cluster.createNamespace(CollectorElement.createCollectorElement(extensionContext.getRequiredTestClass().getName(), extensionContext.getRequiredTestMethod().getName()), namespaceWhereCreationOfCustomResourcesIsApproved);
+        clusterOperator = clusterOperator.defaultInstallation()
+            .withWatchingNamespaces(Constants.WATCH_ALL_NAMESPACES)
+            // use our pre-defined Roles
+            .withRoles(roles)
+            // use our pre-defined RoleBindings
+            .withRoleBindings(roleBindings)
+            // use our pre-defined ClusterRoles
+            .withClusterRoles(clusterRoles)
+            // use our pre-defined ClusterRoleBindings
+            .withClusterRoleBindings(clusterRoleBindings)
+            .createInstallation()
+            .runBundleInstallation();
+
+        // implicit verification that a user is able to deploy Kafka cluster in namespace <example-1>, where we are allowed
+        // to create Custom Resources because of `*-namespaced Role`
+        resourceManager.createResource(extensionContext, false, KafkaTemplates.kafkaEphemeral(testStorage.getClusterName(), 3)
+            .editMetadata()
+            // this should work
+            .withNamespace(namespaceWhereCreationOfCustomResourcesIsApproved)
+            .endMetadata()
+            .build());
```

Comment:
Indent
Also, if it should work, shouldn't we wait for readiness of Kafka?
Or deploy it in parallel and wait for first Kafka to be ready and second Kafka to contain message about failure?

Reply:
> Also, if it should work, shouldn't we wait for readiness of Kafka?

Yes, that's leftover...

> Or deploy it in parallel and wait for first Kafka to be ready and second Kafka to contain message about failure?

Well, I will re-order the deployment of these Kafka's (and we can achieve that kind of parallelism with additional readiness check).